Sources/RSDKUtils/Extensions/UIColor+Extensions.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+/// Extension to `UIColor` class to provide addtional initializers required by InAppMessaging.
```

Comment:
Don't need to mention IAM - these should be generic utils.

Reply:
True, forgot to change that description